### PR TITLE
Clean up and improve UI of track install window

### PR DESCRIPTION
**Pull request by rwjuk**
This PR cleans up both the code for the 'install track design' window, and the UI it generates. I've also moved the track design name to the bottom section (as it was often hard to see overlain on the preview), and added the ride type to the info as well - it's not always obvious what ride a certain track design is!

capture

### CODE REVIEW

File: src/openrct2/windows/install_track.c [hunk 1]
```diff
@@ -246,14 +252,19 @@ static void window_install_track_paint(rct_window *w, rct_drawpixelinfo *dpi)
...
     x = w->x + widget->left + 1;
-    y = w->y + widget->bottom + 2;
+    y = w->y + widget->bottom + 4;
     // 0x006D3CF1 -- 0x006d3d71 missing
 
+    // Track design name & type
+    gfx_draw_string_left(dpi, STR_TRACK_DESIGN_NAME, &_trackName, COLOUR_BLACK, x-1, y);
+    y += 10;
+
+    rct_string_id friendlyTrackName = RideNaming[td6->type].name;
```
Comment (Gymnasiast) [inline src/openrct2/windows/install_track.c]: `get_friendly_track_type_name()`. This will properly return stuff like 'Hyper-Twister'. RideNaming contains the technical track names as used in the Scenario Editor.

Comment (rwjuk) [inline src/openrct2/windows/install_track.c]: I did think about that, but how will I get the `rct_ride_entry` from the TD6?

Comment (Gymnasiast) [inline src/openrct2/windows/install_track.c]: `td6->vehicle_object`

Comment (rwjuk) [inline src/openrct2/windows/install_track.c]: @Gymnasiast But that's an `rct_object_entry`, which doesn't look related to `rct_ride_entry`. I'm probably missing something obvious though

Comment (IntelOrca) [inline src/openrct2/windows/install_track.c]: `rct_ride_entry` is the actual ride entry data, the TD6 does not store that, it only stores a reference to it, `rct_object_entry` same entries used in scenarios.

**Comment by rwjuk** [comment]
CIs failing due to Github outage

File: src/openrct2/windows/install_track.c [hunk 2]
```diff
@@ -261,7 +261,8 @@ static void window_install_track_paint(rct_window *w, rct_drawpixelinfo *dpi)
     gfx_draw_string_left(dpi, STR_TRACK_DESIGN_NAME, &_trackName, COLOUR_BLACK, x-1, y);
     y += 10;
 
-    rct_string_id friendlyTrackName = RideNaming[td6->type].name;
+    sint32 groupIndex = object_manager_get_loaded_object_entry_index(object_manager_load_object(&td6->vehicle_object));
```
Comment (Gymnasiast) [inline src/openrct2/windows/install_track.c]: There is no protection against NULL here. If one loads a track design that uses a vehicle the user does not have, this could crash the game.

Comment (rwjuk) [inline src/openrct2/windows/install_track.c]: Good catch.

**Comment by duncanspumpkin** [comment]
I've found a small bug in the install track button. 
https://github.com/OpenRCT2/OpenRCT2/pull/6080/files#diff-f4aedd34242818f7d806992116275ef8R428

This line is trying to copy the existing file into the existing file. This will always fail. Could you change `_trackPath` to `destPath`.

**Comment by rwjuk** [comment]
@duncanspumpkin What's the line number I'm looking at - the link isn't that clear?

**Comment by duncanspumpkin** [comment]
One sec I feel like im going round in circles trying to work out the source of the issue.

**Comment by duncanspumpkin** [comment]
Okay so my initial assumption was wrong there is a bit of bug in the install button. Its enough of a bug that I'm just going to make it a separate issue.

**Comment by Gymnasiast** [comment]
Just one thing: the correct function is `get_ride_naming()`, not `get_friendly_track_type_name()` as I said earlier.

**Comment by rwjuk** [comment]
@Gymnasiast That returns a `rct_ride_name`, which I'd need to extract the `rct_string_id` from anyway. Is the logic different between the two?

**Comment by Gymnasiast** [comment]
Could you make that change and rebase?
